group2_assignment2_solution.ipynb

```diff
@@ -495,7 +507,10 @@
     "you have crawled for each keyword, save them into a file named w.json where w is the keyword.\n",
```

Comment:
Note that these instructions don't quite match the file names specified in the PDF, which contain our group name.